stories/patterns/filters/advanced-filters/docs/docs-examples/filters-with-filter-conditions.tsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
looks like we don't need this file

Reply:
It's widely used across the file.

Reply:
We don't need this file. Appeared here after merge as duplicate file. Fixed.

Reply:
you still have this file in diff

Reply:
removed